R/metapredict_plot.R

```diff
@@ -192,7 +203,8 @@ plotClassProbsCv = function(cvFit, lambda, ematMerged, sampleMetadata, className
       if (any(df$trueClass==classLevel)) {
         idxTmp = c(idxTmp, 1:(sum(df$trueClass==classLevel)))}}
     df$idx = idxTmp
-    dfMolten = tidyr::gather(df, key='probClass', value='prob', !!classLevels)
+    # dfMolten = tidyr::gather(df, key='probClass', value='prob', !!classLevels)
+    dfMolten = melt(as.data.table(df), measure.vars = classLevels, variable.name = 'probClass', variable.factor = FALSE, value.name = 'prob')
```

Comment:
Unclear why code sometimes uses `as.data.table` and sometimes `data.table` when being passed a data.frame